### PR TITLE
PPTP-1648: Added page

### PR DESCRIPTION
**Pull request by mauro-fanton**
hookup the change link and added more test

### Description of Work carried through

Brief description of what/why/how.

#### Check list 
 - [X] `./precheck` was executed (Integration/Component/Unit tests)
 - [X] `sbt scalafmt test:scalafmt` was executed 
 - [ ] Required Environment Config has been amended/added - N/A
 - [ ] Links to dependencies have been included (BE/FE work) - N/A
 - [X] User Acceptance Tests (UAT) were run locally and have passed.
 - [ ] No Accessibility errors/warnings reported by Wave -N/A

### CODE REVIEW

**Review by PSouthern — COMMENTED** [review]
Do the migrate files need to be in here?

**Comment by mauro-fanton** [comment]
Yes, the migrate file is to do with the scaffolding. I just changed a little to allow to accept a parameter to allow to create route using snake case format. As default this create routes using Camel Case format.

**Comment by mauro-fanton** [comment]
> Yes, the migrate file is to do with the scaffolding. I just changed a little to allow to accept a parameter to allow to create route using snake case format. As default this create routes using Camel Case format.

Actually I reverted the migrate.sh changes, as this should not gone in this PR. This should go inn a separate changes by itself so it can be discussed if it worth it doing.

File: app/models/UnitMeasureType.scala [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+object UnitMeasureType extends Enumeration {
```
Comment (liampan) [inline app/models/UnitMeasureType.scala:19]: as discussed, lets use the viewmodels converter

File: .gitignore [hunk 2]
```diff
@@ -35,3 +35,4 @@ test-result
 tmp
 .idea_modules
 bin
+*.bak
```
Comment (liampan) [inline .gitignore:38]: bark 🐶

File: app/viewmodels/checkAnswers/HumanMedicinesPlasticPackagingWeightSummary.scala [hunk 3]
```diff
@@ -24,7 +24,7 @@ import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
 import viewmodels.govuk.summarylist._
 import viewmodels.implicits._
 
-object HumanMedicinesPlasticPackagingWeightSummary extends SummaryViewModel {
+class HumanMedicinesPlasticPackagingWeightSummary extends SummaryViewModel {
```
Comment (liampan) [inline app/viewmodels/checkAnswers/HumanMedicinesPlasticPackagingWeightSummary.scala:27]: as discussed, lets turn these back to objects

File: app/viewmodels/checkAnswers/ImportedPlasticPackagingSummary.scala [hunk 4]
```diff
@@ -24,14 +24,14 @@ import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
 import viewmodels.govuk.summarylist._
 import viewmodels.implicits._
 
-object ImportedPlasticPackagingSummary extends SummaryViewModel {
+class ImportedPlasticPackagingSummary(key: String ) extends SummaryViewModel {
```
Comment (liampan) [inline app/viewmodels/checkAnswers/ImportedPlasticPackagingSummary.scala:27]: as discussed, lets turn these to private classes, with static objects in the companion

**Review by liampan — CHANGES_REQUESTED** [review]
Looks good! few comments we've discussed :)

File: .gitignore (hunk 2 above)
Comment (mauro-fanton) [reply .gitignore]: removed

File: app/viewmodels/checkAnswers/HumanMedicinesPlasticPackagingWeightSummary.scala (hunk 3 above)
Comment (mauro-fanton) [reply app/viewmodels/checkAnswers/HumanMedicinesPlasticPackagingWeightSummary.scala]: done

File: app/viewmodels/checkAnswers/ImportedPlasticPackagingSummary.scala (hunk 4 above)
Comment (mauro-fanton) [reply app/viewmodels/checkAnswers/ImportedPlasticPackagingSummary.scala]: done

**Review by liampan — APPROVED** [review]

**Review by PSouthern — APPROVED** [review]

**Pull request merged**